asr-python-adapter-finetune-am-conformer-nemo.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://developer.download.nvidia.com/notebooks/dlsw-notebooks/riva_asr_asr-python-advanced-finetune-am-citrinet-tao-finetuning/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# How to customize a Riva ASR Acoustic Model (Conformer) with Adapters\n",
+    "This tutorial walks you through how to customize a Riva ASR acoustic model (Conformer CTC) with Adapter modules, using the NVIDIA NeMo toolkit."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NVIDIA Riva Overview\n",
+    "\n",
+    "NVIDIA Riva is a GPU-accelerated SDK for building speech AI applications that are customized for your use case and deliver real-time performance. <br/>\n",
+    "Riva offers a rich set of speech and natural language understanding services such as:\n",
+    "\n",
+    "- Automated speech recognition (ASR)\n",
+    "- Text-to-Speech synthesis (TTS)\n",
+    "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
+    "\n",
+    "In this tutorial, we will customize a Riva ASR acoustic model (Conformer) with Adapter modules, using the NeMo Toolkit. <br> \n",
+    "To understand the basics of Riva ASR APIs, refer to [Getting started with Riva ASR in Python](https://github.com/nvidia-riva/tutorials/blob/stable/asr-python-basics.ipynb). <br>\n",
+    "\n",
+    "For more information about Riva, refer to the [Riva developer documentation](https://developer.nvidia.com/riva)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Neural Module (NeMo) Toolkit\n",
+    "NVIDIA Neural Module (NeMo) Toolkit is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and natural language understanding (NLU) models with a simple Python interface. Developers, researchers, and software partners building intelligent conversational AI applications and services, can bring their own data to fine-tune pre-trained models instead of going through the hassle of training the models from scratch."
```

Comment:
```suggestion
    "NVIDIA Neural Module (NeMo) Toolkit is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and NLU models with a simple Python interface. Developers, researchers, and software partners building intelligent conversational AI applications and services, can bring their own data to fine-tune pre-trained models instead of going through the hassle of training the models from scratch."
```